### DISCUSSION TITLE
Do we plan on adding TLS 1.3 Early Data  support?

### DISCUSSION THREAD

**Discussion by LaRuim** [curl/curl#13528] [Q&A] [answered]
If that's not on the roadmap yet, is it perhaps something a small team of college students can work on adding as a class project over a couple of months?

**Comment by bagder** [comment, marked as answer]
> is it perhaps something a small team of college students can work on adding as a class project over a couple of months?

I think that sounds like a pretty good idea, yes!

**Comment by LaRuim** [comment]
Thanks! I'll open an ideas post with a plan of action.
